### PR TITLE
Update FAQ with more Questions

### PR DESCRIPTION
**Pull request by rnystrom**
Been compiling a list of more questions, adding them to the FAQ. Note that this includes some 3.0-only stuff.

### CODE REVIEW

File: Guides/Best Practices and FAQ.md [hunk 1]
```diff
@@ -58,14 +62,91 @@ Yes.
 
 No, but you can install the [diffing subspec via CocoaPods](https://instagram.github.io/IGListKit/installation.html).
 
-#### What's the purpose of `IGListCollectionView`?
-
-We use this subclass to gain compile-time safety to prevent disallowed methods from being called on `UICollectionView`, because `IGListKit` handles model and view updates. See discussion at [#409](https://github.com/Instagram/IGListKit/issues/409).
-
```
Comment (jessesquires) [inline Guides/Best Practices and FAQ.md]: we should leave this for historical reasons

**Comment by jessesquires** [comment]
updated inline 😄

**Comment by Sherlouk** [comment]
"When does version 'x' come out"
"Soon" :trollface:

**Comment by jessesquires** [comment]
@Sherlouk LOL

**Pull request closed without merging**